sdk-testing/src/main/java/io/dapr/actors/client/ActorClientStub.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Microsoft Corporation and Dapr Contributors.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.client;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * Stub implementation of ActorClient to enable unit testing in apps.
+ * <p>This class is needed because ActorClient is a class and not an interface.</p>
```

Comment:
This is an incorrect assumption. Mockito allows you to mock Classes as well... as long they aren't _final_.
The reason for that behavior is because **Mockito will create a subclass at runtime** (see screenshot, you can see in the debugger that the instance "mock" is actually an automatically generated subclass of what you are mocking (ActorClient in this case).. So the users of the SDK don't actually need this stub implementation to help them test.

![image](https://user-images.githubusercontent.com/11362563/130503852-57c4a084-4ad0-4e2a-927b-f569bed6e257.png)

Reply:
Awesome! I will update the comment. On the other hand, we still need to make the method public here. I was keeping it package protected to make sure the App does not invoke any method in ActorClient and only used internally.